NOTICE

```diff
@@ -79,33 +79,6 @@ OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 
-Django REST Swagger <https://github.com/marcgibbons/django-rest-swagger>
```

Comment:
Do we keep this file up-to-date?
Should I include `drf-spectacular`?

Reply:
Answer: we don't... I think you can just remove swagger. Maybe we should just include libraries which a (a) important and (b) where we use more than just single functions in the RDMO code.